test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -313,75 +313,67 @@ func TestMachineConfigPoolChangeRestartsBuild(t *testing.T) {
 	require.NoError(t, err)
 }
 
-// This test starts a build with an image that is known to fail because it does
-// not have the necessary binaries within it. After failure, it edits the
-// MachineOSConfig with the expectation that the failed build and its objects
-// will be deleted and a new build will start in its place.
+// This test starts a build with an image that is known to fail because it uses
+// an invalid containerfile. After failure, it edits the  MachineOSConfig
+// with the expectation that the failed build and its  will be deleted and a new
+// build will start in its place.
 func TestGracefulBuildFailureRecovery(t *testing.T) {
```

Comment:
This test has been updated to use a bad container file as suggested in https://github.com/openshift/machine-config-operator/pull/4756#discussion_r1884562915. It now takes about 17 minutes to run locally, not sure how fast it used to be prior to that!